src/components/Text/Text.module.scss

```diff
@@ -0,0 +1,38 @@
+@import "../../assets/variables.scss";
+
+.text {
```

Comment:
you're not using `.text` class in your component 😉

Reply:
☝️ still valid + `font-family: "Montserrat";` could be placed on `body` in `index.scss` and affect all components like `Text`, `Button` etc., (the file could be imported in `App.jsx`)